src/components/index-page/MainPage.tsx

```diff
@@ -4,12 +4,12 @@ export default function LoginPage() {
     return (
         <body style={{ backgroundColor: '#5F6F52' }}>
             <S.MainLogo grid1={40 + '%'} grid2={30 + '%'} grid3={30 + '%'}>
-                <S.MainLogoText margintop={230} fontSize={5}>
+                <S.MainLogoText margintop={230 + '%'} fontSize={5 + '%'}>
```

Comment:
```suggestion
                <S.MainLogoText margintop={`${230}%`} fontSize={5 + '%'}>
```
이런식으로 `벡틱`으로도 처리가 가능합니다.

Reply:
props로 크기를 전달하는 부분들은 
```
'100%'
```
이런식으로 일괄 처리할 예정입니다